packages/documentation/src/content/docs/integration/prod/docker-compose.mdx

```diff
@@ -2,66 +2,315 @@
+| Open Payments auth server     | Exposes a reference implementation of an Open Payments authorization server | auth.DOMAIN  | auth.myrafiki.com  |
+| Admin UI                      | Exposes an Admin UI to manage Rafiki                                        | admin.DOMAIN | admin.myrafiki.com |
+
+</div>
+
+:::note
+The example domain and subdomain values are for demonstration purposes only. You must use the actual domain names that you set up with your DNS host.  
+:::
+
+### VM specifications
+
+A general purpose VM with the following specifications is sufficient for running Rafiki via Docker:
+
+- OS: Linux distro
+- RAM: 4GB
+- vCPUs: 2
+
+### Install Nginx and Certbot
+
+Once you have provisioned the VM in your cloud environment, install Nginx along with Certbot:
 
 ```sh
 sudo apt update && sudo apt install nginx certbot python3-certbot-nginx
 ```
 
-## Domain preparation
+### Domain preparation
 
-Generate the Let’s encrypt certificates:
+Generate the Let’s Encrypt certificates using Certbot:
 
 ```sh
 certbot certonly --manual --preferred-challenges=dns --email EMAIL --server https://acme-v02.api.letsencrypt.org/directory --agree-tos -d DOMAIN
 ```
 
-:::note
+:::caution[Update TXT record]
 Domain can be in wildcard format. You will also need to update the TXT record in this step.
 :::
 
-As Let's encrypt certificates are valid for 90 days, you must set up a cron process to renew the certificate on a regular schedule:
+As Let's Encrypt certificates are valid for 90 days, you must set up a cron process to renew the certificate on a regular schedule:
 
 ```sh
 crontab -e
 ​
-0 3 * * * crontab renew
+0 3 * * * certbot renew
 ```
 
-## Update DNS records
+### Update DNS records
+
+Next update the DNS records (A records) to point to the static external IP address of the virtual machine:
+
+<div class="overflow-table">
+
+| service                       | URL          | example            |
+| ----------------------------- | ------------ | ------------------ |
+| Open Payments resource server | DOMAIN       | myrafiki.com       |
+| ILP Connector                 | ilp.DOMAIN   | ilp.myrafiki.com   |
+| Open Payments auth server     | auth.DOMAIN  | auth.myrafiki.com  |
+| Admin UI                      | admin.DOMAIN | admin.myrafiki.com |
 
-Next update the DNS records to point to the static external IP address of the volumes:
+</div>
+
+:::note
+The example domain and subdomain values are for demonstration purposes only. You must use the actual domain names that you set up with your DNS host.  
+:::
 
-| service   | URL              | example                |
-| --------- | ---------------- | ---------------------- |
-| admin     | admin.DOMAIN     | admin.myrafiki.com     |
-| auth      | auth.DOMAIN      | auth.myrafiki.com      |
-| connector | connector.DOMAIN | connector.myrafiki.com |
-| ilp       | ilp.DOMAIN       | ilp.myrafiki.com       |
+### Install Docker Engine and Docker Compose plugin
```

Comment:
Maybe we can just link to the docker engine install page? since it'll differ per distro 
https://docs.docker.com/engine/install/